libs/api/shared/database/prisma/src/lib/schema.prisma

```diff
@@ -38,13 +38,27 @@ model Session {
 }
 
 model User {
-  id            String    @id @default(cuid())
+  id            String       @id @default(cuid())
   name          String?
-  email         String?   @unique
+  email         String?      @unique
   emailVerified DateTime?
   image         String?
   accounts      Account[]
   sessions      Session[]
+  profile       UserProfile?
+}
+
+model UserProfile {
+  id         String   @id @default(cuid())
+  createdAt  DateTime @default(now())
+  updatedAt  DateTime @updatedAt
+  bannerUrl  String?
+  profileUrl String?
+  pseudo     String   @unique
+  punchline  String?
+  userName   String?
+  User       User     @relation(fields: [userId], references: [id])
```

Comment:
I think you should keep de property name in camel case. Make sure your relations are in camel case too.

```suggestion
  user       User     @relation(fields: [userId], references: [id])
```